CHANGELOG.md

```diff
@@ -7,24 +7,34 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 - Add support for specifying additional extensions to load ([#547], [#563]).
+- Add support for specifying additional extensions to load ([#547], [#563])
+- Support Apache Druid `30.0.0` as new LTS version ([#583])
 
 ### Changed
 
 - Bump `stackable-operator` from `0.64.0` to `0.70.0` ([#585]).
 - Bump `product-config` from `0.6.0` to `0.7.0` ([#585]).
 
-[#585]: https://github.com/stackabletech/airflow-operator/pull/585
-
 ### Fixed
 
 - [BREAKING] Move the DB credentials `user` and `password` out of the CRD into a secret containing the keys `username` and `password` ([#557]).
 - Processing of corrupted log events fixed; If errors occur, the error
   messages are added to the log event ([#572]).
 
+### Changed
```

Comment:
Ok last thing, there is already a `### Changed` header (linter)

Reply:
[`4744215` (#583)](https://github.com/stackabletech/druid-operator/pull/583/commits/474421567c21498a9efdb9e911712f3ff7061c47)

Ah, this came in with #585 in the last hour